config/rbac/kustomization.yaml

```diff
@@ -12,7 +12,7 @@ resources:
 # Comment the following 4 lines if you want to disable
 # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
 # which protects your /metrics endpoint.
-# - auth_proxy_service.yaml
+- auth_proxy_service.yaml
 - auth_proxy_role.yaml
 - auth_proxy_role_binding.yaml
-# - auth_proxy_client_clusterrole.yaml
+- auth_proxy_client_clusterrole.yaml
```

Comment:
@anandf we don't need this uncommented, we have replaced this clusterrole with the role defined in `config/prometheus`

Reply:
ack. Removed all changes done to this file `config/rbac/kustomization.yaml`